packages/apps/plugins/plugin-observability/src/ObservabilityPlugin.tsx

```diff
@@ -193,6 +195,12 @@ export const ObservabilityPlugin = (options: {
               observability.event(event);
               return { data: event };
             }
+
+            case ObservabilityAction.CAPTURE_USER_FEEDBACK: {
+              const feedback = intent.data as UserFeedback;
+
+              observability.captureUserFeedback(feedback.email, feedback.name, feedback.message);
```

Comment:
make sure to return something non-null here otherwise the intent plugin will interpret this as unhandled

Reply:
Do you have a go to return value to just indicate 'handled'?

Reply:
The default I've done elsewhere is just `return { data: true };`